android/app/build.gradle

```diff
@@ -135,6 +135,7 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-photo-view')
```

Comment:
@andrewda great :smile: 

If not against @housseindjirdeh, then I would leave this dependency, it does not seem heavy and does not interfere with the application. 

BTW, I now thought, how much the application will increase in size after connection of any dependency (or it becomes more slow to work for example)? What are thoughts?